src/Utilities/Utilities.jl

```diff
@@ -444,54 +427,48 @@ end
 url(remote, repo, doc) = url(remote, repo, doc.data[:module], doc.data[:path], linerange(doc))
 
 # Correct file and line info only available from this version onwards.
-if VERSION >= v"0.5.0-dev+3442"
-    function url(remote, repo, mod, file, linerange)
-        remote = getremote(dirname(file))
-        isabspath(file) && isempty(remote) && isempty(repo) && return Nullable{Compat.String}()
-        # Replace any backslashes in links, if building the docs on Windows
-        file = replace(file, '\\', '/')
-        # Format the line range.
-        line = format_line(linerange)
-        # Macro-generated methods such as those produced by `@deprecate` list their file as
-        # `deprecated.jl` since that is where the macro is defined. Use that to help
-        # determine the correct URL.
-        if inbase(mod) || !isabspath(file)
-            base = "https://github.com/JuliaLang/julia/tree"
-            dest = "base/$file#$line"
-            Nullable{Compat.String}(
-                if isempty(Base.GIT_VERSION_INFO.commit)
-                    "$base/v$VERSION/$dest"
-                else
-                    commit = Base.GIT_VERSION_INFO.commit
-                    "$base/$commit/$dest"
-                end
-            )
-        else
-            commit, root = cd(dirname(file)) do
-                toplevel = readchomp(`git rev-parse --show-toplevel`)
-                if in_cygwin()
-                    toplevel = readchomp(`cygpath -m "$toplevel"`)
-                end
-                readchomp(`git rev-parse HEAD`), toplevel
-            end
-            if startswith(file, root)
-                if isempty(repo)
-                    repo = "https://github.com/$remote/tree/{commit}{path}#{line}"
-                end
-
-                _, path = split(file, root; limit = 2)
-                repo = replace(repo, "{commit}", commit)
-                repo = replace(repo, "{path}", path)
-                repo = replace(repo, "{line}", line)
-
-                Nullable{Compat.String}(repo)
+function url(remote, repo, mod, file, linerange)
+    remote = getremote(dirname(file))
+    isabspath(file) && isempty(remote) && isempty(repo) && return Nullable{Compat.String}()
```

Comment:
This could just be `String` rather than `Compat.String`. (Same in another couple places.) It should be possible to ditch the Compat dependency entirely in this package, but I'm not certain.